frontend/src/libs/components/prompt-generation/libs/components/prompt-card/styles.module.scss

```diff
@@ -0,0 +1,45 @@
+.container {
+  min-width: 224px;
+}
+
+.prompt {
+  height: 73px;
+  padding: 29.61px 5px 21.39px;
+  overflow: auto;
+  text-align: center;
+  background-color: var(--white);
+  border-radius: 5px;
+}
+
+.category {
+  position: relative;
+  z-index: 10;
```

Comment:
Move it to variables, but I'm not sure if you need additional indexing

Reply:
It was required for scroll animation, I will remove it then.